midp/localmsg.js

```diff
@@ -18,6 +18,15 @@ var LocalMsgConnection = function() {
     this.clientMessages = [];
 }
 
+LocalMsgConnection.prototype.reset = function() {
+    this.clientWaiting = [];
+    this.clientMessages = [];
+    while (this.serverWaiting.length > 0) {
+        this.serverWaiting.shift()(false);
+    }
+    this.serverMessages = [];
+}
```

Comment:
This doesn't reset `clientConnected` or `waitingForConnection`. I'm not familiar with this class so maybe that's by design?